src/GameList.elm

```diff
@@ -11,18 +13,22 @@ import Json.Encode exposing (Value)
 
 
 type GameName
```

Comment:
It took awhile, but I'm so happy that there's only one file that needs to be changed outside of your code now to add a new gmae.